toqito/state_opt/bell_inequality_max.py

```diff
@@ -0,0 +1,168 @@
+"""Computes the upper bound for a given Bipartite bell inequality."""
+
+from itertools import combinations
+
+import cvxpy as cp
+import mosek
+import numpy as np
+from scipy.sparse import eye
+
+from toqito.channels import partial_transpose
+from toqito.perms import permutation_operator, swap
+
+
+def MN_matrix(m: int, a: int, x: int) -> np.ndarray:
+    r"""Compute the matrices M_a^x and N_b^y.
+
+    Args:

```

Comment:
Hi! Please take a look at the 'contributing' section of the documentation. We have a specific style for docstrings params etc. 

https://toqito.readthedocs.io/en/latest/contributing.html#references-in-docstrings

Reply:
Also, you need to add unit tests too to check the function you added works as expected. 

Are you a beginner to open-source? I can provide some links on how to write unit tests.

Reply:
You can likely look at any of the existing `test/` directories in the `toqito` project to see how unit tests are written as well (as this might be more instructive for writing tests in this case).

Reply:
@EthanFeld See if this helps you: https://third-bit.com/py-rse/testing.html

>  I wasn't sure if I should add it because its just a helper function

If you want this helper function to be available in the API then you should make sure everything about this is properly documented. Otherwise, use an `_` in front of the function name so that it will be a private function. 

https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-single-and-double-underscore-before-an-object-name